src/scenes/PostThumbnail/PostThumbnail.js

```diff
@@ -6,23 +6,15 @@ import classNames from 'classnames';
 
 import styles from './PostThumbnail.module.scss';
 
-const PostThumbnail = ({
-  title,
-  slug,
-  featured_media,
-  categories,
-  isFirst,
-}) => {
+const PostThumbnail = ({ title, slug, featured_media, categories }) => {
   const imageSource = featured_media.source_url;
   const imageOptimized =
     featured_media.localFile &&
     featured_media.localFile.childImageSharp &&
     featured_media.localFile.childImageSharp.fluid;
   const link = `/blog/${slug}`;
 
-  const articleClass = classNames(styles.container, {
-    [styles.firstArticle]: isFirst,
-  });
+  const articleClass = classNames(styles.container);
 
   return (
     <article className={articleClass}>
```

Comment:
we can remove line 17 and replace this line to:
```suggestion
    <article className={styles.container}>
```

Reply:
@lazio исправил